src/kixi/search/elasticsearch.clj

```diff
@@ -1,15 +1,16 @@
 (ns kixi.search.elasticsearch
   (:require [cheshire.core :as json]
-            [clojurewerkz.elastisch.native :as esr]
-            [clojurewerkz.elastisch.native
-             [document :as esd]
-             [response :as esrsp]]
             [clj-http.client :as client]
+            [clojure.spec.alpha :as spec]
+            [clojurewerkz.elastisch.native :as esr]
+            [clojurewerkz.elastisch.native.document :as esd]
+            [com.rpl.specter :as specter]
             [environ.core :refer [env]]
             [joplin.repl :as jrepl]
+            [kixi.search.query-model :as model]
             [kixi.search.time :as t]
-            [taoensso.timbre :as timbre :refer [error info]]
-            [com.rpl.specter :as specter]))
+            [medley.core :refer [map-vals]]
```

Comment:
how much are you using out of medley? If it is only 1-3 the suggestion w/libraries like this is to copy out what you need rather than bring in the dependency to avoid guava like issues.